subsys/bluetooth/controller/Kconfig.df

```diff
@@ -3,18 +3,27 @@
 # Copyright (c) 2020 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-# BT_CTLR_DF_SUPPORT is a wrapper for all DF features.
-# If only certain DF features are supported, those should be selected
-# individually.
-config BT_CTLR_DF_SUPPORT
+# BT_CTLR_DF_SUPPORT is a wrapper for all DF features that are supported by
+# Zephyr's BLE Controller.
+config BT_LL_SW_SPLIT_DF_SUPPORT
```

Comment:
config BT_LL_SW_SPLIT_DF_SUPPORT is valid only for BT_LL_SW_SPLIT. 
Should it be moved to Kconfig.bt_ll_sw_split?

Reply:
yeah, good point. Thank you!

Reply:
I've changed the PR according to @cvinayak  suggestion so your concern is also addressed.